### PR TITLE
Adding library ComandLineParser to manage command line args

### PR DESCRIPTION
**Pull request by amcandio**
+Added library: https://github.com/commandlineparser/commandline

### CODE REVIEW

File: AnalyzeDotNetProject.csproj [hunk 1]
```diff
@@ -1,12 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
+
```
Comment (npazosmendez) [inline AnalyzeDotNetProject.csproj]: Please delete trailing whitespaces. Review the [contributing guidelines](www.contributing..com.ar) before sending a PR.

**Comment by thepalbi** [comment]
Nice. Thanks for the collaboration @amcandio 🎉🎉🎉!

**Review by thepalbi — APPROVED** [review]
Please fix @npazosmendez comment

**Comment by thepalbi** [comment]
Don't forget to cherry pick the PRs

**Pull request merged**